sync/user_management.go

```diff
@@ -54,9 +54,9 @@ func GetUsedUserIDs() map[int]bool {
 }
 
 // GetFreeUserID returns the smallest valid unused user id
-func GetFreeUserID() int {
+func GetFreeUserID() int64 {
 	used := GetUsedUserIDs()
-	for i := 0; i <= math.MaxInt64; i++ {
+	for i := int64(0); i <= math.MaxInt64; i++ {
```

Comment:
nit: we could also use `math.MaxInt`, to prevent using `int64` on 32 systems.

Reply:
I think it is better to make `userID` a consistent type across all systems so that data can be migrated from system to system with different `int` sizes. This way, data from a 64-bit system with user IDs larger than 32 bits can be reused in a 32-bit system.

Reply:
Fine for me. Storage seems to use `int` user ids as well:
https://github.com/timewarrior-synchronize/timew-sync-server/blob/ca5623596cf3eaaca7d629e822971f8c08d5cfd9/storage/storage.go#L29